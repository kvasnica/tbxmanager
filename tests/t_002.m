function t_002
% "tbxmanager show sources" should return just the default

onCleanup(@(x) tbx_setupTest('done')); tbx_setupTest('start');

% full syntax
tbxmanager show sources

% short syntax
tbxmanager sh so

end
