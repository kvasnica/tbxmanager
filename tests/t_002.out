--- tbxmanager show sources 
Active sources:

http://www.tbxmanager.com/package/index.xml
--- tbxmanager sh so 
Active sources:

http://www.tbxmanager.com/package/index.xml
